### DISCUSSION TITLE
Topaz for Electron Microscopy?

### DISCUSSION THREAD

**Discussion by declined000** [tbepler/topaz#202] [Q&A]
Hi:) I'm a complete beginner but do you guys think I could also apply topaz for denoising electron microscopy data not just cryoEM? All my files are in .tif format.  I'm working on alignment of EM to match LM data, so far my EM data are just 2D images of 8 slices of brain but I'm attempting to create a 3D reconstruction from alignment recordings (each 225 images) to match the LM 3D stack but before I do that I want to make sure all images in EM are very readable so I can use it for further 3D stacking. Alternatively if you would have any other tips/models I could use I would highly appreciate it:)

**Comment by tbepler** [comment]
I haven't personally ever tried that before, but it would be pretty straightforward for you to feed your images into the pre-trained denoising model and see how it works.

Do you have frames for each image? If you do, you could use those to create a dataset to train a new denoising model for your EM images.

**Reply by declined000** [reply]
Yes, so for each of 8 planes of brain I have alignment files for them consisting of 225 frames where the researcher was basically trying to find the sweet spot to acquire best quality image going above and bellow the point they wanted to capture which makes it possible to assemble all of them into one 3 dimensional stack of 8 x 225 frames but denoising is crucial

thank you:) in this case I will try to feed it in pre-trained model and see or alternatively train a new model, one concern I might have is the data format as cryoEM usually comes in .mrc and I have .tif but lets see
